dotnet-471-dev-pack/test/test.ps1

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env pwsh
+
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Validate Input Parameters and System Software
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+param (
+    [Parameter(Mandatory=$true)]
+    [string]$PackageIdentifier = "",
+    [ValidateScript({
+        if( -Not ($_ | Test-Path) ){
+            throw "File does not exist"
+        }
+        if($_ -notmatch "(\.hart)"){
+            throw "The file specified in the path argument must be of type *.hart"
+        }
+        return $true
+    })]
+    [System.IO.FileInfo]$PathToHartArtifact
+)
+if(-Not ( $PathToHartArtifact | Test-Path) ){
+    throw "-PathToHartArtifact must a string and set to a valid path to a *.hart artifact"
+}
+
+# validate pre-requisite software on test system
+if (-Not (Get-Module -ListAvailable -Name Pester)) {
+    $message = -join @(
+        "Pester module must be installed before proceeding. "
+        "Try something like the following and then re-run: "
+        "'Install-Module Pester -SkipPublisher -Force'"
+    )
+    throw $message
+}
+
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# MAIN
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+# install habitat package
+hab pkg install $PathToHartArtifact
```

Comment:
Pending us agreeing on the [test design](https://github.com/habitat-sh/core-plans/issues/2473) - the package should already be installed and the tests should just take the pkg ident as a parameter.
Since that design isn't ratified yet, that shouldn't block this PR, but we'd likely need to come back and change this later.

Reply:
Fixed.  Only $pkg_ident passed as parameter.  test.ps1 assumes that plan has already been built and installed.  README updated accordingly as well.